frontend/src/components/LogInPopup/LogInPopup.jsx

```diff
@@ -0,0 +1,69 @@
+import * as Dialog from "@radix-ui/react-dialog";
+import { Link } from "react-router-dom";
+import "./LogInPopup.scss";
+
+function LogInPopup() {
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger asChild>
+        <button type="button" className="navbarLink">
+          S'identifier
+        </button>
+      </Dialog.Trigger>
+      <Dialog.Portal>
+        <Dialog.Overlay className="dialogOverlay" />
+        <Dialog.Content className="dialogContent">
+          <Dialog.Title className="dialogTitle">
+            Connexion à mon espace
+          </Dialog.Title>
+          <Dialog.Description className="dialogDescription">
+            Déjà inscrit ? Connectez-vous.
+          </Dialog.Description>
+          <fieldset className="fieldset">
+            <label className="label" htmlFor="UserName">
+              Nom d'utilisateur
```

Comment:
It's confusing to ask for the name but wait for the email.

Reply:
solved